proto/beacon/rpc/v1_gateway/services.pb.go

```diff
@@ -0,0 +1,2216 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
```

Comment:
do we need this ? not sure if its needed

Reply:
it's nice to have at least for some native Go support for people that use it, we can leave it. The comments are added automatically by proto-gen-go